syftlib/src/main/java/org/openmined/syft/domain/JobRepository.kt

```diff
@@ -27,6 +29,17 @@ internal class JobRepository(
     val status: DownloadStatus
         get() = trainingParamsStatus.get()
 
+    fun copyDiffScriptAsset(config: SyftConfiguration): String {
+        if (!File(config.filesDir, DIFF_SCRIPT_NAME).exists())
+            jobLocalDataSource.save(
+                config.context.assets.open("torchscripts/$DIFF_SCRIPT_NAME"),
+                config.filesDir.toString(),
+                DIFF_SCRIPT_NAME
+            ).blockingGet()
```

Comment:
Why `blockingGet`?

Reply:
I have to wait till the copying actually finishes for further tasks. Better is to take a single out of it. I will do it

Reply:
one thing that seems better is to have another save function that does this in realtime